src/civic/index.js

```diff
@@ -476,46 +480,12 @@ const downloadEvidenceRecords = async (url, trustedCurators) => {
             counts.error++;
             continue;
         }
-
-        if (
-            record.significance === 'NA'
-            || (record.significance === null && record.evidenceType === 'PREDICTIVE')
```

Comment:
No, they still can, but we now handle that logic in https://github.com/bcgsc/pori_graphkb_loader/blob/develop/src/civic/relevance.js since https://github.com/bcgsc/pori_graphkb_loader/pull/112 and according to discussion in https://www.bcgsc.ca/jira/browse/KBDEV-1015.

For instance, Evidence Items with a Significance of 'NA':
- will be given a Relevance of 'likely oncogenic' if of Type 'oncogenic';
- will be given a Relevance of  'likely predisposing' if of Type 'predisposing';
- otherwise the Evidence Item will not be processed into a GKB Statement.